app/react-components/Gene.js

```diff
@@ -168,39 +168,39 @@ let Gene = (() => {
               disease_type: c.project.disease_type,
               cancer_type: c.project.cancer_type || 'tbd',
               site: c.project.primary_site,
-              cases: cases,
+              cases,
               ssms: [...new Set([...(acc[c.project.project_id] || { ssms: [] }).ssms, ...c.ssm.map(m => m.ssm_id)])],
-              freq: cases.length/allCasesAggByProject[c.project.project_id],
-            }
+              freq: cases.length / allCasesAggByProject[c.project.project_id],
+            },
           };
         }, {});
 
       const sortedCancerDistData = Object.keys(cancerDistData)
-        .map(k => ({project_id: k, ...cancerDistData[k]}))
+        .map(k => ({ project_id: k, ...cancerDistData[k] }))
         .sort((a, b) => b.freq - a.freq);
 
       const totalNumCases = Object.keys(allCasesAggByProject).reduce((sum, b) => sum + allCasesAggByProject[b], 0);
 
       const frequentMutations = fm.map(x => {
-        let consequence = x.consequence.find(x => x.transcript.is_canonical);
+        const consequence = x.consequence.find(x => x.transcript.is_canonical);
         return {
           ...x,
           num_affected_cases_all: x.occurrence.length,
           num_affected_cases_by_project: x.occurrence.reduce((acc, o) => ({
             ...acc,
-            [o.case.project.project_id]: acc[o.case.project.project_id] ? acc[o.case.project.project_id] + 1 : 1
+            [o.case.project.project_id]: acc[o.case.project.project_id] ? acc[o.case.project.project_id] + 1 : 1,
           }), {}),
           impact: consequence.transcript.annotation.impact,
           consequence_type:
-            <span>
-              <b>{_.startCase(consequence.transcript.consequence_type.replace('variant', ''))}</b>
-              <span style={{marginLeft:'5px'}}>
-                <a href={`/genes/${consequence.transcript.gene.gene_id}`}>{consequence.transcript.gene_symbol}</a>
-              </span>
-              <span style={{marginLeft:'5px', color: impactColors[consequence.transcript.annotation.impact] || 'inherit'}}>
-                {consequence.transcript.aa_change}
-              </span>
-            </span>
+  <span>
```

Comment:
The fix for this is to use parens. I'm touching this PR up with these kinds of things

```

         key: (
          <div />
         )
```